### ISSUE TITLE
Wznawianie oferty - nowy wymagany parametr na kategorii

### ISSUE THREAD

**Issue by bmusialekcepd** [allegro/allegro-api#9638]
Witam, 
załóżmy przypadek ze zakladam nowa oferte po czym wyczerpuje sie jej stan i przez jakis czas wisi jako nieaktywna. Po jakims czasie chce wznowic oferte. Moze byc przypadek ze dla danej ofert doszedl nowy wymagany parametr i nie bede mogl wystawic wznowic takiej oferty?

**Comment by Lukasz-Zurek** [comment]
Tak, może się zdarzyć taka sytuacja. Warto korzystać w tej sytuacji z zasobu [GET /sale/offers/unfilled-parameters](https://developer.allegro.pl/documentation/#operation/getOffersUnfilledParametersUsingGET_1). Więcej informacji znajdziesz w [poradniku](https://developer.allegro.pl/tutorials/jak-zarzadzac-ofertami-7GzB2L37ase#jak-sprawdzic-nieuzupelnione-parametry-w-ofertach).
